notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>1st paragraph, 1st sentence:</p><p>"Before you send image data to AI Platform, encode it using <a href="https://docs.python.org/3/library/base64.html" target="_blank">base64</a>. This makes binary data safe from modification while it is transferred over the Internet."</p><p> </p><p>4th paragraph:</p><p>"When you save the model..."</p><p> </p><p>In the description of each signature: "Overrides..." "Tells..." "Tells..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>